src/main/resources/db/migration/db.feedback.migration/V202409231113__create_company_and_employee_tables.sql

```diff
@@ -0,0 +1,17 @@
+CREATE TABLE company
+(
+    id BIGSERIAL PRIMARY KEY,
+    name VARCHAR(255) NOT NULL
+);
+
+
+CREATE TABLE employee (
+    id BIGSERIAL PRIMARY KEY,
+    first_name VARCHAR(255) NOT NULL,
+    last_name VARCHAR(255) NOT NULL,
+    role VARCHAR(50) CHECK (role IN ('admin', 'manager', 'employee')) NOT NULL,
+    company_id BIGINT REFERENCES company(id),
+    employee_id INT NOT NULL unique,
```

Comment:
what's that?

Reply:
unique - I can add constraint over one column to be unique.

Reply:
and it should be unique...

Reply:
the id (first field) is the employeeId